jetty:run does not work in Eclipse in npm mode on Mac

```
[ERROR] Failed to determine 'node' tool.
[ERROR] Please install it either:
[ERROR]   - by following the https://nodejs.org/en/download/ guide to install it globally
[ERROR]   - or by running the frontend-maven-plugin goal to install it in this project:
[ERROR]   $ mvn com.github.eirslett:frontend-maven-plugin:LATEST:install-node-and-npm -DnodeVersion=v11.6.0 
```

I have it installed globally but Flow relies on it being available in PATH, which is not the case with Eclipse

--- a/flow-server/src/main/java/com/vaadin/flow/server/frontend/FrontendToolsLocator.java
+++ b/flow-server/src/main/java/com/vaadin/flow/server/frontend/FrontendToolsLocator.java
@@ -22,7 +22,6 @@ import java.io.InputStreamReader;
 import java.io.Serializable;
 import java.nio.charset.StandardCharsets;
 import java.util.Arrays;
-import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.TimeUnit;
@@ -72,7 +71,12 @@ public class FrontendToolsLocator implements Serializable {
     public Optional<File> tryLocateTool(String toolName) {
         return executeCommand(isWindows() ? "where" : "which", toolName)
                 .map(this::omitErrorResult).map(CommandResult::getStdout)
-                .orElse(Collections.emptyList()).stream().map(File::new)
+                // Add most common paths in unix #5611
+                .orElse(Arrays.asList(
+                        "/usr/local/bin/" + toolName,
+                        "/opt/local/bin/" + toolName,
+                        "/opt/bin/" + toolName))
+                .stream().map(File::new)
                 .filter(this::verifyTool).findFirst();
     }
 

